init-tools.sh

```diff
@@ -144,24 +145,20 @@ if [ ! -e $__INIT_TOOLS_DONE_MARKER ]; then
 
         echo "Initializing BuildTools..."
         echo "Running: $__BUILD_TOOLS_PATH/init-tools.sh $__scriptpath $__DOTNET_CMD $__TOOLRUNTIME_DIR" >> $__init_tools_log
+
+        chmod +x $__BUILD_TOOLS_PATH/init-tools.sh
```

Comment:
I think these "chmod" hacks should have links to the underlying issue:  [https://github.com/NuGet/Home/issues/4424](https://github.com/NuGet/Home/issues/4424)